penguin/tools/browser_harness_ownership.py

```
@@ -0,0 +1,158 @@
+"""Ownership persistence for browser-harness daemon identities."""
+
+from __future__ import annotations
+
+import datetime
+import json
+import os
+import tempfile
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Any, Iterator
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)